soap/views.py

```
@@ -0,0 +1,92 @@
+from .services import SOAPReportGenerator, generate_markdown
+
+
+User = get_user_model()
+
+
+class GenerateSOAPView(APIView):
+    """
+    API برای تولید گزارش SOAP
+    """
+
+    permission_classes = [IsAuthenticated, IsDoctor]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)